var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -40,11 +39,10 @@ class PyPillow(PythonPackage):
 
     # Required dependencies
     depends_on('binutils', type='build', when=sys.platform != 'darwin')
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
-
-    # Recommended dependencies
-    depends_on('jpeg', when='+jpeg')
-    depends_on('zlib', when='+zlib')
+    depends_on('jpeg')
+    depends_on('zlib')
```

Comment:
These dependencies used to be optional, but that's no longer the case. I removed the corresponding variants.